services/chronikService.ts

```diff
@@ -434,10 +453,12 @@ export class ChronikBlockchainClient implements BlockchainClient {
     }
     for (const transaction of blockTxsToSync) {
       const addressesWithTransactions = await this.getAddressesForTransaction(transaction)
+      const inputAddresses = transaction.inputs.map(inp => outputScriptToAddress(this.networkSlug, inp.outputScript))
```

Comment:
why not use the function you created here? Why sorted input addresses elsewhere but not here?